extide/gradle/src/org/netbeans/modules/gradle/ActionProviderImpl.java

```diff
@@ -500,14 +503,12 @@ Action forProject(Project p, FileObject fo) {
     private final class CustomPopupActions extends AbstractAction implements Presenter.Popup {
 
         private final boolean onFile;
-        private final boolean onGradle;
         private final Lookup lookup;
 
         @SuppressWarnings("OverridableMethodCallInConstructor")
         private CustomPopupActions(boolean onFile, boolean onGradleFile, FileObject fo) {
```

Comment:
is `onGradleFile` parameter used ?

Reply:
Well it is not used, but calculated earlier. This one is a copy+paste left over from the maven plugin. Probably have to check why we calculate that on the first place, also check that in the "original" maven plugin code as well.